articles/file-handling-in-Cpp/index.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/file-handling-in-Cpp/
+title: File Handling in C++
+description: In this article, we will walk through how to implement the File Handling function in C++, and we will cover the key working pointers of file handling.
+author: parampreet-singh
+date: 2020-09-21T00:00:00-12:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/file-handling-in-Cpp/hero.jpg
+    alt: functions Function C++ cpp
+---
+If you are a Software Engineer, you may come across File handling at some point in your career. It is the ability to store the output and perform various operations on it, such as updating the data or overwriting things in a file. Files are always a vital way to store data.
+<!--more-->
+### Why do you need File Handling?
+- **Reusability**: Generally, developers work with big data, and it's always not efficient to write everything out. But by storing data in files, one can use it anytime.
```

Comment:
everything out. But by storing data in files, ->  everything out. By storing data in files,

Reply:
can use it anytime.

what is 'it'?